CMakeLists.txt

```diff
@@ -31,6 +31,8 @@ option(HIDE_NON_EXTERNAL_SYMBOLS
     "Hide all non EXTERNAL tagged symbols" ON)
 option(ENABLE_STRICT_COMPILER_SETTINGS
     "Enable compiler flags to get the warnings/errors due to improper condition in the code" OFF) 
+option(LEGACY_CONFIG_GENERATOR
```

Comment:
Does this mean that if I pass no parameter, it uses the legacy one ? I assume the new one works with the old *and* new cmake versions and as been tested, so my proposal would be to set this defautlt to OFF, do not use the legacy stuff unless specifically requested.

Reply:
That would require all existing plugin configs to be converted to use new config generator. This is a pending ticket in team's backlog. Currently, only a few plugins uses the new generator.

Reply:
I forgot to update the description... If no parameter is passed the current behavior is honored. That means that the <Plugin.config> is used by default and the all the needed CMake files are installed. 

If switched off It will require the plugin to have a new <Plugin>.conf.in template. This way we have an option to move in the background to the new stuff.